src/views/homepage/index.tsx

```diff
@@ -48,7 +48,7 @@ const HomePageMobileMenu = () => {
 
 const HomePageContent = () => {
   return (
-    <>
+    <div className={s.homepageContent}>
```

Comment:
We need the mobile menu to stack above `homepageContent`, class here to make that possible.